site/content/docs/manifest/lb-web-service.en.md

```diff
@@ -259,6 +293,9 @@ A sidecar container that requests are routed to instead of the main service cont
 If the target container's port is set to `443`, then the protocol is set to `HTTPS` so that the load balancer establishes 
 TLS connections with the Fargate tasks using certificates that you install on the target container.
 
+<span class="parent-field">http.</span><a id="http-target-port" href="#http-target-port" class="field">`target_port`</a> <span class="type">String</span>  
+The container port that receives traffic. Specify this field if the container port is different from `iamge.port` for the main container or `sidecar.port` for the sidecar containers.
```

Comment:
nit: what do you think of something like this ⬇️ ? My main nitpick is to invite `target_container` into this description. Not sure if this is necessary given the `target_` prefix though! Lmk what you think:

```suggestion
Optional. The container port that receives traffic. By default, this will be `image.port` if the `target_container` is the main container, or `sidecars.<name>.port` if the `target_container` is a sidecar.
```

Reply:
I like the idea, the only concern I have is 
1. Customer shouldn't think that `target_container` is required considering `target_port`'s value depends on `target_container`. We can solve this by setting `target_container` description to start with "Optional ".
2. This new statement is true only when target_container is specified by the customer in case of main container. Because even if customer doesn't provide `target_container` then we set it as main container. So is it safe to display conditional statements here? I am not sure if I am making sense. Do let me know what do you think. Because I like your idea anyway.

Reply:
For the second point, then maybe we can do this:

`Optional. The container port that receives traffic. By default, this will be `image.port` if the target container is the main container, or `sidecars.<name>.port` if the `target_container` is a sidecar.`

Instead of saying `target_container`, we say "target container", i.e. not the field name.